build.sbt

```diff
@@ -545,6 +545,7 @@ lazy val docs = project
   .enablePlugins(DocusaurusPlugin)
 
 lazy val javaOnlySettings = List[Def.Setting[_]](
+  javafmtOnCompile := false,
```

Comment:
This was doing my head in – IntelliJ was formatting one way, running SBT session (~snapshots/run) another way, and IJ constantly producing a warning to load filesystem changes.

Formatting on compile is evil.

Reply:
> Formatting on compile is evil.

Agreed.